validation/README.md

```diff
@@ -4,6 +4,10 @@ This directory contains a JSON schema to validate OSV entries.
 
 ## Usage
 
+```
```

Comment:
Should add a sentence that this can be used with any JSON schema validator, and that these are two examples?

Reply:
Clarified in [6784904](https://github.com/ossf/osv-schema/pull/241/commits/67849046d932afcd37117c0fdb6a156afec9e665)